__tests__/app.test.js

```diff
@@ -470,4 +470,30 @@ describe('11. GET /api/reviews/:review_id (comment count)', () => {
             })
         })
     })
-})
\ No newline at end of file
+})
+
+describe('12. DELETE /api/comments/:comment_id', () => {
+    test('status:204, responds with an empty response body', () => {
+      return request(app).delete('/api/comments/2').expect(204);
```

Comment:
We could add a test to make sure there is no response body. For example expect(!body)